lib/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ if(NOT ANDROID)
     endforeach()
 endif()
 
-target_link_libraries(isaac "dl")
+target_link_libraries(isaac "dl" OpenCL pthread)
```

Comment:
should remove pthread here. The reason why we need pthread is the clBLAS we build with pthread support.